src/app-layout/__tests__/common.test.tsx

```diff
@@ -308,7 +288,9 @@ describeEachAppLayout(size => {
     });
 
     test('Close button does have a label if it is defined', () => {
-      const { wrapper } = renderComponent(<AppLayout contentType="form" {...(singleDrawerOpen as any)} />);
+      const { wrapper } = renderComponent(
+        <AppLayout activeDrawerId={singleDrawer[0].id} drawers={singleDrawer} onDrawerChange={() => {}} />
```

Comment:
Here and below, I am setting an empty function to suppress controllability warning and have more clean console when inspecting test logs